packages/fluttium_cli/lib/src/commands/new_command/new_bundle_command.dart

```diff
@@ -44,6 +44,7 @@ class NewBundleCommand extends Command<int> {
       switch (props.type) {
         case BrickVariableType.enumeration:
         case BrickVariableType.array:
+        case BrickVariableType.list:
```

Comment:
That comes from a newer version of mason right @felangel?

We should probably bump the Mason dependency to match the new code

Reply:
Updated the `mason` version to `^0.1.0-dev.50`

Reply:
Yup that’s correct